CaloCluster/fcl/prolog_trigger.fcl

```diff
@@ -41,6 +44,41 @@ CaloClusterFast :
 }
 
 
+CaloClusterCompare : {
+	module_type : CaloClusterCompare
+        CaloClusterCollection : NewCaloClusterFast
+  }
+
+
+NewCaloClusterFast :
+{
+  module_type            : CaloProtoClusterFromCrystalHit
+  caloCrystalModuleLabel : FastCaloCrystalHitFromHit
+  mainClusterCollName    : None
+  splitClusterCollName   : None
+  isOnline               : true
+  minEnergy              : 50
+  EminSeed               : 20 
+  ExpandCut              : 1 
+  EnoiseCut              : 1
+  timeCut                : @local::HitMakerBlindTime   
+  deltaTime              : 5.01
+  diagLevel              : 0
+}  
+
+
+NewCaloClusterFast :
```

Comment:
I would move the modules instances within this block in the already present CaloClusterTrigger block.
Also, the reco sequence that uses NewCaloClusterFast should include the sequence from CaloReco/fcl/prolog_trigger.fcl that performs the hit reco (starting from the digi collection.